app/panel/components/BuildingBlocks/CliqzFeature.jsx

```diff
@@ -0,0 +1,124 @@
+/**
+ * Cliqz Features Component
+ *
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2019 Ghostery, Inc. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+import React from 'react';
+import ClassNames from 'classnames';
+import Tooltip from '../Tooltip';
+
+/**
+ * @class Implements rendering and interaction for Cliqz feature icon toggles
+ * @memberof PanelBuildingBlocks
+ */
+class CliqzFeature extends React.Component {
+	constructor(props) {
+		super(props);
+
+		this.clickCliqzFeature = this.clickCliqzFeature.bind(this);
+	}
+
+	/**
+	 * Handles clicks on the Cliqz feature icon, toggling it on/off
+	 */
+	clickCliqzFeature() {
+		const {
+			active,
+			clickButton,
+			cliqzInactive,
+			type
+		} = this.props;
+
+		if (cliqzInactive) {
+			return;
+		}
+
+		const featureType = type === 'anti_track' ? 'anti_tracking' : type;
+
+		clickButton({
+			feature: `enable_${featureType}`,
+			status: active,
+			text: this._getAlertText(active, type),
+		});
+	}
+
+	_getStatus(active) {
+		return active ? t('drawer_on') : t('drawer_off');
```

Comment:
I used `drawer_on` and `drawer_off` because they were already in the `locales.json` file.  Should we make better names that are unique to CliqzFeature?

Reply:
Now using `cliqz_feature_status_on` and `cliqz_feature_status_off`